doajtest/unit/test_query.py

```diff
@@ -30,7 +31,12 @@
             "auth" : True,
             "role" : "admin",
             "dao" : "portality.models.Journal"
-        }
+        },
+        "suggestion" : {
```

Comment:
Can we call this `application` instead of `suggestion` - we're trying to slowly remove that terminology from the codebase.

Reply:
I think we need to change at all places if we want to change suggestion to application as this is index_type. Let me know if I need to change at all places.

Reply:
You are quite right, leave as is for now